security/scute/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                scute
+
+version             1.7.0
+revision            0
+categories          security
+maintainers         {@gouttegd dgouttegattat:incenp.org}
+description         PKCS#11 interface to the GnuPG Agent
+long_description    Scute is a PKCS#11 implementation relying on the GnuPG Agent. \
+                    It enables the use of an OpenPGP smartcard for TLS client     \
+                    authentication and for CMS digital signatures with Mozilla    \
+                    and other PKCS#11-compatible applications.
+homepage            https://gnupg.org/
+platforms           darwin
```

Comment:
`platforms darwin` is now implicit in MacPorts, and no longer needs to be specified in Portfiles. You can delete this line.